src/components/Messaging/MessageBody.js

```diff
@@ -370,61 +371,69 @@ const SenderInformation = ({
       </Grid>
       <Grid item className={senderItem}>
         <Typography variant="h5">
-          {type === 'survey'
+          {type === 'survey' || type === 'survey_response'
             ? `Survey: ${message?.survey?.title}`
             : type === 'announce'
             ? `Announcement`
             : messageRecipient}
         </Typography>
 
-        {(type === 'survey' || type === 'announce') && (
+        {(type === 'survey' ||
+          type === 'survey_response' ||
+          type === 'announce') && (
           <>
-            <Typography>
+            <Typography align="left" color="primary">
               <b>DATE:</b> {dateFormat(message?.composed_at, 'yyyy/mm/dd')}
             </Typography>
             {message?.bulk_message_recipients && (
-              <Typography>
-                <b>TO:</b> {message?.bulk_message_recipients[0]?.recipient}
-              </Typography>
-            )}
-            {type === 'survey' && (
-              <Typography>
-                <b>RESPONSES:</b> {responseCount}
+              <Typography align="left" color="primary">
+                <b>TO:</b>{' '}
+                {message?.bulk_message_recipients.length
+                  ? message?.bulk_message_recipients[0]?.recipient
+                  : message?.recipient_organization_id
+                  ? message?.recipient_organization_id
+                  : message?.recipient_region_id}
```

Comment:
Can this be extracted to a method? It's a bit confusing with all the conditions.

Reply:
referenced this in a new issue.